### PR TITLE
zh-Cn: tutorials/keras/basic_text_classification_with_tfhub.ipynb

### PR DESCRIPTION
**Pull request by Berrywrq**

### CODE REVIEW

File: site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "35jv_fzP-llU"
+   },
+   "source": [
+    "## 训练模型\n",
+    "\n",
+    "在 mini-batche 大小为 512 个样本的条件下迭代 20 个 epoch 来训练模型。 这是指对 `x_train` 和 `y_train` 张量中所有样本的的 20 次迭代。在训练过程中，监测来自验证集的 10,000 个样本上的损失值（loss）和准确率（accuracy）："
```
Comment (gaoljhy) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: 原文
Train the model for 20 epochs in mini-batches of 512 samples

建议翻译为
训练一个小批量的512个样本模型迭代20次

File: site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5KggXVeL-llZ"
+   },
+   "source": [
+    "## 进一步阅读\n",
+    "\n",
+    "有关使用字符串输入的更一般方法，以及对训练期间准确率（accuracy）和损失值（loss）变化进程更详细的分析，请参阅[此处](https://www.tensorflow.org/tutorials/keras/basic_text_classification)。"
```
Comment (gaoljhy) [inline site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: 原文
For a more general way to work with string inputs and for a more detailed analysis of the progress of accuracy and loss during training, take a look here.

翻译 准确率（accuracy）和损失值（loss）变化进程
变化进程是?? 建议去掉变化进程

File: site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb (hunk 1 above)
Comment (Berrywrq) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: Thanks for reviewing.  I've made some modifications to your opinion except this one.  Since `mini-batch` and `epoch` are proper nones in machine learning, I think they should be retained and not translated. And `mini-batches of 512 samples` means every training step has 512 samples but not train the model with just 512 samples, for this reason, I hope to retain the previous translation. We can discuss further to determine the final translation result.

Comment (Berrywrq) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: Or `以 512 个样本的 mini-batch 大小迭代 20 个 epoch 来训练模型。`. If there are other suggestions, please tell me. And thanks again.

Comment (gaoljhy) [reply site/zh-cn/beta/tutorials/keras/basic_text_classification_with_tfhub.ipynb]: nothing , you are right

**Comment by gaoljhy** [comment]
lgtm

**Comment by gaoljhy** [comment]
@Berrywrq i don't konw why Conflict is this file - `site/zh-cn/beta/tutorials/quickstart/advanced.ipynb`

**Comment by Berrywrq** [comment]
@gaoljhy Please check if there is still a conflict about advaced.ipynb now
